vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/data/Json.kt

```diff
@@ -28,3 +30,18 @@ val jsonSerializer by lazy {
         }
     }
 }
+
+val Base64UrlStrict = Base64(config = Base64ConfigBuilder().apply {
+    lineBreakInterval = 0
+    encodeToUrlSafe = true
+    isLenient = true
+    padEncoded = false
+}.build())
+
+
+val Base64Strict = Base64(config = Base64ConfigBuilder().apply {
+    lineBreakInterval = 0
+    encodeToUrlSafe = false
+    isLenient = true
+    padEncoded = true
+}.build())
```

Comment:
Why is it not used everywhere?

Reply:
Isn't this covered in https://github.com/a-sit-plus/kmm-vc-library/pull/18 ?

Reply:
Sorry, that was very misleading and sloppy! Why don't we want such a shorthand also for Base16?

But come to think of it: Kotlin 1.9 ships with base64 and hex encoding. We might not need the library any more